cli/azd/cmd/package.go

```diff
@@ -154,11 +154,14 @@ func (pa *packageAction) Run(ctx context.Context) (*actions.ActionResult, error)
 			return nil, err
 		}
 
-		pa.console.StopSpinner(ctx, stepMessage, input.StepDone)
+		if packageResult.FromCached {
+			pa.console.StopSpinner(ctx, stepMessage, input.StepSkipped)
+		} else {
+			pa.console.StopSpinner(ctx, stepMessage, input.StepDone)
+			// report package output
+			pa.console.MessageUxItem(ctx, packageResult)
+		}
```

Comment:
Looks like you were attempting to remove the double package progress output.  The command should not know or be aware that any sort of caching has happened.